### PR TITLE
Feat/playground

### PR DESCRIPTION
**Pull request by tarun-khanna**
# Description
1. i18nify-playground package added in i18nify mono repo. 
2. i18nify playground updated with latest i18nify version and all new APIs

## Changes Made

List the main changes made in this pull request.

| Title                  | --- |
| ---------------------- | --- |
| JIRA link              | NA  |
| Slack thread (if any)  | NA  |
| Product spec           | NA  |
| Tech spec/One-pager    | NA  |
| Bundle Size Difference | NA  |
| Fixes Issue            | NA  |

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] This change is a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Is your change backward compatible ?
- [ ] Tested on major browsers (Chrome, Firefox, Safari, IE) ?
- [ ] Tested in a consumer application(s) ?

## Additional Notes

Any additional information that would be helpful for the reviewer.

## Checklist:

- [ ] Add Jira ID(s) in PR title and in the description?
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My code is written in TypeScript and follows the coding standards of the project.
- [ ] I have added relevant documentation and updated the README if necessary.
- [ ] My commit messages are clear and follow the project's commit message conventions.
- [ ] Is any external library added?
- [ ] My changes do not introduce any new runtime errors or warnings.
- [ ] All relevant unit tests have been added/updated in my PR for the change.
- [ ] Any Screenshots (mobile & desktop) required for PR? If yes, have you added the respective screenshots ?
- [ ] Any manual dev testing done by you on beta/func environment?
- [ ] Reviewer added (SLA 2 days)

## Reviewer Checklist

- [ ] Sufficient QA/Dev-Testing is done with proof (test cases list)
- [ ] Ensure that the change in bundle size falls within the acceptable range.

---

## PR Title Format

Format: `<type>: <subject>`

Types can be as follows:

- `feat`: (new feature for the user, not a new feature for build script)
- `fix`: (bug fix for the user, not a fix to a build script)
- `docs`: (changes to the documentation)
- `style`: (formatting, missing semi colons, etc; no production code change)
- `refactor`: (refactoring production code, eg. renaming a variable)
- `test`: (adding missing tests, refactoring tests; no production code change)
- `chore`: (updating grunt tasks etc; no production code change)

### CODE REVIEW

File: packages/i18nify-playground/public/assets/world_rotating.gif
Comment (RgnDunes) [inline packages/i18nify-playground/public/assets/world_rotating.gif:1]: I don't see it being used. Can we remove this if not being used? 

Screenshot 2024-07-24 at 3 50 05 PM

File: packages/i18nify-playground/src/app.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import 'src/global.css';
+
+import { useScrollToTop } from 'src/hooks/use-scroll-to-top';
+
+import Router from 'src/routes/sections';
+import ThemeProvider from 'src/theme';
+// import Footer from './layouts/footer';
```
Comment (RgnDunes) [inline packages/i18nify-playground/src/app.jsx:7]: remove this commented code. Not required.

File: packages/i18nify-playground/src/app.jsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+// ----------------------------------------------------------------------
+
+export default function App() {
+  useScrollToTop();
+
+  return (
+    <ThemeProvider>
+      <Router />
+      {/* <Footer /> */}
```
Comment (RgnDunes) [inline packages/i18nify-playground/src/app.jsx:17]: remove this commented code. Not required.

File: packages/i18nify-playground/src/constants/locale/index.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  {
+    value: 'en-US',
+    label: 'USA',
+    icon: '/assets/icons/ic_flag_us.svg',
+  },
+  {
+    value: 'ms-MY',
+    label: 'MY',
+    icon: '/assets/icons/ic_flag_my.svg',
+  },
```
Comment (RgnDunes) [inline packages/i18nify-playground/src/constants/locale/index.js:31]: Can we also include some other unique geographies like Baharain or so here or maybe include all supported geo's here? Thinking from an open-source pov.

File: packages/i18nify-playground/src/layouts/dashboard/config-navigation.jsx [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+  //   children: [
+  //     { title: 'GetState', path: '/state/getState' },
+  //     { title: 'SetState', path: '/state/setState' },
+  //   ],
+  // },
+  // {
+  //   title: 'Plugins',
+  //   path: '/plugins',
+  //   icon: icon('ic_plugin'),
+  // },
```
Comment (RgnDunes) [inline packages/i18nify-playground/src/layouts/dashboard/config-navigation.jsx:105]: Can we remove this if we are not using these

File: packages/i18nify-playground/src/layouts/dashboard/header.jsx [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+import Toolbar from '@mui/material/Toolbar';
+import { useTheme } from '@mui/material/styles';
+import IconButton from '@mui/material/IconButton';
+
+import { useResponsive } from 'src/hooks/use-responsive';
+
+import { bgBlur } from 'src/theme/css';
+
+import Iconify from 'src/components/iconify';
+import SvgColor from 'src/components/svg-color';
```
Comment (RgnDunes) [inline packages/i18nify-playground/src/layouts/dashboard/header.jsx:15]: `SvgColor` is not being used anywhere, remove this import.

File: packages/i18nify-playground/src/layouts/dashboard/header.jsx [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+
+import { NAV, HEADER } from './config-layout';
+import LanguagePopover from './common/language-popover';
+// import GitHubIcon from '@mui/icons-material/GitHub';
```
Comment (RgnDunes) [inline packages/i18nify-playground/src/layouts/dashboard/header.jsx:19]: `GitHubIcon` is not being used anywhere, remove this import

File: packages/i18nify-playground/src/pages/home.jsx [hunk 7]
```diff
@@ -0,0 +1,211 @@
...
+                <Link
+                  component="button"
+                  underline="none"
+                  onClick={() => navigate('/plugins')}
+                  color="#2950DA"
+                >
+                  FormatNumberByParts
+                </Link>
+              </Box>
+            </Grid> */}
```
Comment (RgnDunes) [inline packages/i18nify-playground/src/pages/home.jsx:188]: remove

File: packages/i18nify-playground/src/routes/sections.jsx [hunk 8]
```diff
@@ -0,0 +1,184 @@
...
+
+// Date Time Pages
+import DatePage from 'src/pages/date';
+import FormatDateTime from 'src/pages/date/formatDateTime';
+import GetRelativeTime from 'src/pages/date/getRelativeTime';
+import GetWeekdays from 'src/pages/date/getWeekdays';
+import ParseDateTime from 'src/pages/date/parseDateTime';
+
+import ComingSoon from 'src/components/comingSoon';
+import GetMaskedPhoneNumber from 'src/pages/phone/getMaskedPhoneNumber';
```
Comment (RgnDunes) [inline packages/i18nify-playground/src/routes/sections.jsx:44]: move this import in Phone number pages group to keep consistency

File: packages/i18nify-playground/src/constants/locale/index.js (hunk 3 above)
Comment (tarun-khanna) [reply packages/i18nify-playground/src/constants/locale/index.js:31]: yep, we can plan this in next iteration. 
We can include a big list of locales in the dropdown

**Review by RgnDunes — APPROVED** [review]

**Pull request merged**